action.yml

```diff
@@ -0,0 +1,107 @@
+name: 'Conan.io GitHub Action'
```

Comment:
@czoido This will be name in the market place, please, take a look if it looks good.

Reply:
This is a bit generic, I would use something like: `Setup Conan Client` ?

Reply:
Done, changed to `Setup Conan Client`